common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExecutorDiskUtils.java

```diff
@@ -50,14 +58,18 @@ public static File getFile(String[] localDirs, int subDirsPerLocalDir, String fi
    * the internal code in java.io.File would normalize it later, creating a new "foo/bar"
    * String copy. Unfortunately, we cannot just reuse the normalization code that java.io.File
    * uses, since it is in the package-private class java.io.FileSystem.
+   *
+   * On Windows, separator "\" is used instead of "/".
+   *
+   * "\\" is legal character in path name on Unix like OS, but illegal on Windows.
```

Comment:
Maybe,
- `is legal character` -> `is a legal character`.
- `Unix like` -> `Unix-like`.

Reply:
Changed.